lib/galaxy/managers/users.py

```diff
@@ -613,6 +613,8 @@ def send_reset_email(self, trans, payload, **kwd):
 
     def get_reset_token(self, trans, email):
         reset_user = get_user_by_email(trans.sa_session, email, self.app.model.User)
+        if reset_user.deleted:
```

Comment:
```suggestion
        if reset_user and reset_user.deleted:
```
because user could be none